libs/gi/ui/src/components/LevelSelect.tsx

```diff
@@ -53,33 +51,42 @@ export function LevelSelect({
     else setBoth({ ascension: lowerAscension })
   }, [setBoth, ascensionMaxLevels, ascension, level])
   return (
-    <ButtonGroup sx={{ bgcolor: (t) => t.palette.contentNormal.main }}>
-      <CustomNumberInputButtonGroupWrapper>
-        <CustomNumberInput
-          onChange={setLevel}
-          value={level}
-          startAdornment="Lv. "
-          disabled={disabled}
-          inputProps={{
-            min: 1,
+    <Box sx={{ display: 'flex', flexWrap: 'wrap', rowGap: '4px'}}>
+      <TextField
```

Comment:
> Can you implement using TextFieldLazy? https://github.com/frzyc/genshin-optimizer/blob/master/libs/common/ui/src/components/TextFieldLazy.tsx

I can see that you've made a commit today for adding a `NumberInputLazy` component - would it not be better to have that merged into master and then use that?

Reply:
I agree, I was waiting to see which PR goes in first, since my PRs tend to have a delay in review. Looks like its merged in, to master now, you can merge master into your branch and see if `NumberInputLazy` component makes your implementation easier.

Reply:
> since my PRs tend to have a delay in review
I wonder who this is subtweeting

Reply:
:clueless: